react/features/filmstrip/middleware.web.js

```diff
@@ -120,7 +127,8 @@ MiddlewareRegistry.register(store => next => action => {
         const { dispatch, getState } = store;
         const { participantId, pinned } = action;
         const state = getState();
-        const { activeParticipants } = state['features/filmstrip'];
+        const { activeParticipants, maxStageParticipants } = state['features/filmstrip'];
+        const maxNoOfParticipants = Math.min(MAX_ACTIVE_PARTICIPANTS, maxStageParticipants);
```

Comment:
can maxStageParticipants ever be > than `MAX_ACTIVE_PARTICIPANTS`

Reply:
No. I moved this check on SET_MAX_STAGE_PARTICIPANTS to make sure that I can't